src/wp-content/themes/twentyfourteen/inc/featured-content.php

```diff
@@ -439,9 +439,9 @@ public static function enqueue_scripts() {
 	}
 
 	/**
-	 * Get featured content settings.
+	 * Gets featured content settings.
 	 *
-	 * Get all settings recognized by this module. This function
+	 * Gets all settings recognized by this module. This function
```

Comment:
It's part of the description, so IMO we can leave it as is. WDYT @SergeyBiryukov ?